contracts/WildcardSteward_v2.sol

```diff
@@ -193,7 +227,7 @@ contract WildcardSteward_v2 is Initializable {
             price[tokenId]
                 .mul(now.sub(timeLastCollected[tokenId]))
                 .mul(patronageNumerator[tokenId])
-                .div(patronageDenominator)
+                .div(1000000000000)
```

Comment:
any reason for 12 instead of 18?

Reply:
We are already live with that value (10^12), besides that no real logic.

We plan to make a new repo for public consumption without the warts that wildcards has developed from iterating in production; I'll set that to 18 there :)